.github/workflows/linters.yml

```diff
@@ -0,0 +1,36 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  eslint:
+    name: ESLint
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup ESLint
+        run: |
+          npm install --save-dev eslint@6.8.x eslint-config-airbnb-base@14.1.x eslint-plugin-import@2.20.x babel-eslint@10.1.x
```

Comment:
- [ ] I see you have a problem with fixing linters on GHA, the reason is that you have `"es2021": true` in your `.eslintrc.js`, and ESLint in the GHA doesn't recognize it, it's most likely that you're missing a package that needs to be installed in order for it to do so, add the package(s) to this line (line 19) on `linters.yml` for it to work.